src/components/select-menu/SelectMenu.chromatic.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import * as SelectMenuStories from './SelectMenu.stories.mdx';
+import {generateChromaticStory} from '../../chromatic/utils';
+import SelectMenu, {SIZE as SELECT_SIZE} from './SelectMenu';
+
+import Text from '../text/Text';
+import Flex from '../flex/Flex';
+
+const onOptionChange = () => null;
+
+export const Sizes = args => {
+  return (
+    <div style={{display: 'flex', flexDirection: 'row'}}>
+      {Object.values(SELECT_SIZE).map(size => (
+        <div style={{width: '200px'}} key={size}>
+          <SelectMenu
+            {...args}
+            defaultExpanded
+            size={size}
+            onOptionChange={onOptionChange}
+          />
+        </div>
+      ))}
+    </div>
+  );
+};
+
+export const DifferentPopupLenghts = args => {
+  return (
+    <Flex direction="row" style={{gap: '40px'}}>
+      <div style={{width: '200px'}}>
+        <Text>Popup with wide content:</Text>
+        <SelectMenu
+          options={[
+            {
+              value: 'tv',
+              label: 'Usłyszałem o Brainly w reklamie telewizyjnej w telewizji',
```

Comment:
Why PL lang?

Reply:
It was used in the figma file, but I can totally change it to eng

Reply:
I see, please ask designer to change it accordingly in figma. Our base lang is always eng. We should only use different lang if we prepare something specific to one market.